subgraphs/openzeppelin-governor/protocols/ens-governance/src/ens-governor.ts

```diff
@@ -0,0 +1,108 @@
+import { Address } from "@graphprotocol/graph-ts";
+import {
+  ProposalCanceled,
+  ProposalCreated,
+  ProposalExecuted,
+  ProposalQueued,
+  QuorumNumeratorUpdated,
+  TimelockChange,
+  VoteCast,
+} from "../../../generated/ENSGovernor/ENSGovernor";
+import {
+  _handleProposalCreated,
+  _handleProposalCanceled,
+  _handleProposalExecuted,
+  _handleProposalQueued,
+  _handleVoteCast,
+} from "../../../src/handlers";
+import { ENSGovernor } from "../../../generated/ENSGovernor/ENSGovernor";
+import { GovernanceFramework } from "../../../generated/schema";
+
+// ProposalCanceled(proposalId)
+export function handleProposalCanceled(event: ProposalCanceled): void {
+  _handleProposalCanceled(event.params.proposalId.toString(), event);
+}
+
+// ProposalCreated(proposalId, proposer, targets, values, signatures, calldatas, startBlock, endBlock, description)
+export function handleProposalCreated(event: ProposalCreated): void {
+  // FIXME: Prefer to use a single object arg for params
+  // e.g.  { proposalId: event.params.proposalId, proposer: event.params.proposer, ...}
+  // but graph wasm compilation breaks for unknown reasons
+
+  _handleProposalCreated(
+    event.params.proposalId.toString(),
+    event.params.proposer.toHexString(),
+    event.params.targets,
+    event.params.values,
+    event.params.signatures,
+    event.params.calldatas,
+    event.params.startBlock,
+    event.params.endBlock,
+    event.params.description,
+    event
+  );
+}
+
+// ProposalExecuted(proposalId)
+export function handleProposalExecuted(event: ProposalExecuted): void {
+  _handleProposalExecuted(event.params.proposalId.toString(), event);
+}
+
+// ProposalQueued(proposalId, eta)
+export function handleProposalQueued(event: ProposalQueued): void {
+  _handleProposalQueued(event.params.proposalId, event.params.eta);
+}
+
+// QuorumNumeratorUpdated(oldQuorumNumerator, newQuorumNumerator)
+export function handleQuorumNumeratorUpdated(
+  event: QuorumNumeratorUpdated
+): void {
+  let governanceFramework = getGovernanceFramework(event.address.toHexString());
```

Comment:
Is there a reason you didn't pull out these smaller handlers into the common handlers file?

Reply:
theyre tied to governanceFramework that imports/bind the contracts